src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2532,6 +2532,389 @@ void spiral(index_t ndoms,
     }
 }
 
+//---------------------------------------------------------------------------//
+void venn_full_matset(Node &res,
+                      index_t nx,
+                      index_t ny)
+{
+    // create the material sets
+
+    index_t elements = nx * ny;
+
+    res["matsets/matset/topology"] = "topo";
+    res["matsets/matset/volume_fractions/background"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_a"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_b"] = DataType::float64(elements);
+    res["matsets/matset/volume_fractions/circle_c"] = DataType::float64(elements);
+
+    float64_array cir_a = res["fields/circle_a/values"].value();
+    float64_array cir_b = res["fields/circle_b/values"].value();
+    float64_array cir_c = res["fields/circle_c/values"].value();
+
+    float64_array mat_bg = res["matsets/matset/volume_fractions/background"].value();
+    float64_array mat_ca = res["matsets/matset/volume_fractions/circle_a"].value();
+    float64_array mat_cb = res["matsets/matset/volume_fractions/circle_b"].value();
+    float64_array mat_cc = res["matsets/matset/volume_fractions/circle_c"].value();
+    
+    for(index_t idx = 0; idx < elements; idx++)
+    {
+        mat_ca[idx] = cir_a[idx];
+        mat_cb[idx] = cir_b[idx];
+        mat_cc[idx] = cir_c[idx];
+        mat_bg[idx] = 1. - (cir_a[idx] + cir_b[idx] + cir_c[idx]);
+    }
+}
+
+void make_sparse(Node & src, index_t len, Node & n)
+{
+    float64_array src_val = src.value();
+
+    index_t nz = 0;
```

Comment:
The variable `nz` contains the number of non-zero entries in the data array being made sparse, but its name can be interpreted to convey the opposite (e.g. "**n**um**z**eroes" instead of "**n**on**z**eroes"). In the interest of clarify, could we rename this variable to something like `num_sparse` or `nsparse`?

Reply:
I went with `nsparse`.